parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -91,6 +91,34 @@ ensureAuditD() {
   fi
 }
 
+{{- if ShouldConfigTransparentHugePage}}
+configureTransparentHugePage() {
+    THP_CONFIG_PATH="/etc/sysfs.conf"
```

Comment:
super nit: this is not just for THP right? so should call it ETC_SYSFS_CONF_PATH or similar

Reply:
updated to ETC_SYSFS_CONF